python/tutorials/05-layer-norm.py

```diff
@@ -35,7 +35,7 @@ def _layer_norm_fwd_fused(X, Y, W, B, M, V, stride, N, eps,
     # compute std
     xmean = tl.where(mask, x - mean, 0.)
     var = tl.sum(xmean * xmean, axis=0) / N
-    rstd = 1 / tl.sqrt(var + eps)
+    rstd = 1 / tl.libdevice.sqrt(var + eps)
```

Comment:
I think we should keep `tl.sqrt` as a base function. We can map it to tl.libdevice.sqrt internally.